<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addChoices</class>
 <widget class="QMainWindow" name="addChoices">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="text">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>691</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Butadd">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add a choice</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Butfinish">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>540</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Finish</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Intxt1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>270</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Inac1">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>280</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="Intxt2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>320</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Inac2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>330</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="Intxt3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>370</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Inac3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>380</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="Intxt4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>420</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Inac4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>430</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="Intxt5">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>470</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Inac5">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>480</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
